### PR TITLE
Add Capybara Session#evaluate_async_script support

### PR DESCRIPTION
**Pull request by twalpole**
This implements support for Capybaras upcoming `Session#evaluate_async_script`.  It also adds a sleep to a test that failed to pass (popen3 hanging) without it on OSX 10.12.6 - Couldn't figure out exactly why.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-sudo: required
+sudo: false
```
Comment (jferris) [inline .travis.yml]: Is this required from the other changes?

File: src/EvaluateAsync.cpp [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+void EvaluateAsync::start() {
+  QString script = arguments()[0];
+  QString jsonArgs;
+  if (arguments().length()>1){
+    jsonArgs = arguments()[1];
+  } else {
+    jsonArgs ="[]";
+  }
+  QString eval_script = QString("(function(){"
```
Comment (jferris) [inline src/EvaluateAsync.cpp]: There's a `capybara.js` file that's injected onto every page. Is it possible to add this function in there?

File: .travis.yml (hunk 1 above)
Comment (twalpole) [reply .travis.yml]: Whoops - nope, it was a leftover from when Travis `sudo: required` builds were broken the other night.

File: src/EvaluateAsync.cpp (hunk 2 above)
Comment (twalpole) [reply src/EvaluateAsync.cpp]: Probably, but that wasn't how src/Evaluate.cpp was written and I'm not 100% sure whether there are any special permissions allowed for JS scripts loaded by Qt (as opposed to scripts loaded by the visited page).  If there are special permissions then having an accessible `eval` may not be great.

Comment (jferris) [reply src/EvaluateAsync.cpp]: Ah, right - we have to inject this differently because we're interpolating JavaScript code in there.

**Pull request merged**